.github/pr-title-checker-config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "LABEL": {
+    "name": "title needs [type]",
+    "color": "EEEEEE"
+  },
+  "CHECKS": {
+    "prefixes": ["[BugFix] ", "[Enhancement]", "[Feature]", "[Refactor]", "[UT]", "[Doc]"],
+    "regexp": "\\[.*\\].*",
+    "regexpFlags": "i",
+    "ignoreLabels" : ["ignore-pr-title-check", "pass"]
+  },
+  "MESSAGES": {
+    "success": "All OK",
+    "failure": "Add type in the title of the pull request. e.g. [type] content",
```

Comment:
Can we refine the error message to let them know what title types we currently support?

Reply:
> Can we refine the error message to let them know what title types we currently support?

actually we support prefix with type in "[]", BugFix, Enhancement, Doc, UT are just recommended type.
this is just a reminder, in case there are other types, like [actions], [WIP], or others